Makefile

```diff
@@ -147,12 +167,23 @@ ifeq ($(TAG_LATEST), true)
 endif
 	@docker images -q $(REGISTRY)/fsfreeze-pause:$(VERSION) > .container-$(DOTFILE_IMAGE)
 
-all-containers:
-	$(MAKE) container
-	$(MAKE) container BIN=velero-restic-restore-helper
-	$(MAKE) build-fsfreeze
+manifest-fsfreeze: BIN = fsfreeze-pause
+manifest-fsfreeze:
+	@docker manifest create $(MULTIARCH_IMAGE):$(VERSION) \
+		$(foreach arch, $(MANIFEST_PLATFORMS), $(MULTIARCH_IMAGE)-$(arch):$(VERSION))
+	@docker manifest push $(MULTIARCH_IMAGE):$(VERSION)
+ifeq ($(TAG_LATEST), true)
+	docker manifest create $(MULTIARCH_IMAGE):latest \
+		$(foreach arch, $(MANIFEST_PLATFORMS), $(MULTIARCH_IMAGE)-$(arch):latest)
+	docker manifest push $(MULTIARCH_IMAGE):latest
+endif
+
+#all-containers:
```

Comment:
These `all-containers` and `all-push` targets should be removed if we don't need them.

Reply:
`all-containers` - will build images for each of the supported arch `CONTAINER_PLATFORMS := linux-amd64 linux-ppc64le`

`all-push` - will push all of this arch images

This will be followed by `all-manifest`
which will create a manifest out of this arch images and push it.

Reply:
I'm specifically talking about the commented out blocks here; they should be removed as the definition changed.

Reply:
Done